src/app/view/components/dot-theme-selector-dropdown/dot-theme-selector-dropdown.component.html

```diff
@@ -0,0 +1,15 @@
+<dot-searchable-dropdown
+    [(ngModel)]="selectedTheme"
+    (change)="onChange($event)"
+    (filterChange)="handleFilterChange($event)"
+    [placeholder]="placeholder"
+    [data]="themes"
+    [totalRecords]="totalRecords"
+    (pageChange)="handlePageChange($event)"
+    rows="5"
+    labelPropertyName="name"
+    overlayWidth="350"
+    valuePropertyName="name"
+>
+    <!-- TODO: change overlayWidth to value + unit before it's merged -->
```

Comment:
I think this change is already in master, if you merge master into this branch you should be good to do this change.